docs/OpConventions.md

```diff
@@ -0,0 +1,12 @@
+# Operator Conventions
+
+To maintain consistency in operator signatures, we use the following principles:
+- All attribute names should be lower case and use underscores when it helps with readability
+- Any input/output represented by a single letter is capitalized (i.e. X)
+- Any input/output represented by a full word or multiple words is all lower case and uses underscores when it helps with readability
+- Any input/output representing a bias tensor will utilize the name "B"
+- Any input/output representing a weight tensor will utilize the name “W”
+- “axes” is used when an input, output or attribute is representing multiple axes
+- “axis” is used when an input, output or attribute is representing a single axis
+- Default type for integers will use variants on int, no uints, default is int64
+- Any instance of multiplying a weight tensor by input tensor and adding bias will following the format of W*X^T+B
```

Comment:
I don't understand this line (the ^T part). Is this some common notation?

Reply:
Other things look good to me.

Reply:
It actually should be `X*W^T+B`. Might make sense to clarify the usual shapes of inputs (e.g. X being `batch X a` and W being `b X a`)